ansible/roles/events/tasks/main.yaml

```diff
@@ -0,0 +1,145 @@
+- include: ../../common/tasks/setfacts.yml
+  tags:
+    - docker
+    - update-images
+    - events
+    - service
+    - docker-config
+    - events_vhost
+
+- name: Add Docker GPG apt Key
```

Comment:
Just a comment. I think that this docker part merits to be moved to a reusable new `docker` role (as tomcat/nginx/mysql/etc). This way we can reuse it.